string obs
float64 pose_x
float64 pose_y
float64 vel_x
float64 vel_y